source/docs/advanced/payments/custom-payment-information.md

```diff
@@ -0,0 +1,138 @@
+---
+id: custom-payment-information
+title: Custom Payment Information
+---
+
+Implemented payment methods don't need any additional information. The user only need to select them and he will be prompted to enter all needed information. When implementing a payment method however you may require more information (Like you may want to let users enter a comment related to the payment method/checkout).
```

Comment:
> The user only need to select them and he

This is not super clear, who/what is `them` referring to? 


I rephrased the ending a bit, although I think it could be improved, first need context around my question above.
```suggestion
Implemented payment methods don't need any additional information. The user only needs to select them and he will be prompted to enter all the required information. However, you may require more information, for example, if you want to let users enter a comment related to the payment method/checkout.
```